Source/Scene/Cesium3DTileset.js

```diff
@@ -787,6 +799,7 @@ define([
         var length = children.length;
         for (var i = 0; i < length; ++i) {
             var child = children[i];
+            child.updateTransform(); // Need to update transform and bounding volumes before checking distance
```

Comment:
This is OK to me...no concern about the L1/L2 cache implications of either approach right now (but maybe later when we use bitfields so we can get more tiles into a cache line).

Just rename `computeDistanceToCamera` to something like `updatePerFrameTileProperties`.